qcodes/instrument/group_parameter.py

```diff
@@ -221,7 +221,10 @@ def set(self, set_parameter: GroupParameter, value: Any) -> None:
         """
         if any((p.get_latest() is None) for p in self.parameters.values()):
             self.update()
-        calling_dict = {name: p.raw_value
+        for _, p in self.parameters.items():
+            print(f'while setting raw value{p.cache.raw_value} and type'
```

Comment:
remove print statement?

Reply:
yes, stupid mistake. Thanks for catching